lib/api/web-element/element-locator.js

```diff
@@ -24,11 +24,21 @@ class ElementLocator {
     });
   }
 
-  static getOrDefault(obj, prop, defaultValue) {
-    // eslint-disable-next-line no-prototype-builtins
-    const isDefined = Utils.isObject(obj) && obj.hasOwnProperty(prop) && Utils.isDefined(obj[prop]);
```

Comment:
This could also be fixed by just a single line change here:

Just need to replace `obj.hasOwnProperty(prop)` with `(prop in obj)`.

Reply:
But doing so causes other problems because if `__index` in `Element` is `undefined`, `index` property will resolve to `NaN` and hence `this.index` here would be set to `NaN` which will cause issues.